README.md

```diff
@@ -1,5 +1 @@
 # Messaging App
-
-## Database Information
-
-Database URI is in .env variable.
```

Comment:
We do want the readme to include any information someone would need to run the database on their own

Reply:
Definitely agree, have added better instructions to install locally.